Dockerfile

```diff
@@ -1,12 +1,10 @@
-# hug depends on distutils which has been removed in python 3.12
-# so we are stuck with 3.11 for now
 FROM python:3.11-slim
 
 WORKDIR /dracor-metrics
 
 COPY . .
-RUN pip install poetry && poetry install
+RUN pip install poetry && poetry install --without dev
 
 EXPOSE 8030
 
-CMD [ "poetry", "run", "hug", "-f", "app/main.py", "-p", "8030" ]
+CMD [ "poetry", "run", "uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8030" ]
```

Comment:
This is not ideal yet. But it works.

Reply:
Not ideal because of `0.0.0.0`?

Reply:
Sorry, I should have elaborated more.
The FastAPI docs recommend using uvicorn in combination with gunicorn. But so far this is working.

https://fastapi.tiangolo.com/deployment/server-workers/

Reply:
We can probably do it as a follow up.